lib/route.js

```diff
@@ -28,12 +28,9 @@ module.exports = function(api, resourceName, actionName, controller, baseContext
       debug('resource response', response);
       if (err) {
         debug('resource error', err);
-        const message = ('string' === typeof err) ? err : err.message;
         res.caughtError = err;
-        res.json(
-          err.statusCode || 500,
-          { type: err.type || 'Error', message }
-        );
+        const { code, body } = errorTransform(err);
```

Comment:
Should we be at all concerned about maybe receiving an`errorTransform` function that just explodes rather than returning a transformed error?

Reply:
🤷‍♂ I figure thats on the developer to write a safe function to hand it

Reply:
should we await it to allow people to use promises if needed?

Reply:
I'd prefer not to, this is supposed to be a nice simple static transform